### DISCUSSION TITLE
Code example Filter & Pagination

### DISCUSSION THREAD

**Discussion by 6be709c0** [cloudscape-design/components#111] [Q&A] [answered]
Hi,

Is it possible to share the code for https://cloudscape.design/examples/react/table.html.
![image](https://user-images.githubusercontent.com/9129639/182472688-2ab46a94-b9c7-4be6-848b-ae28e5cd93e3.png)

I cannot have the same view.

In the AppLayout, under contentHeader if I put a `TextFilter` and a `Pagination` it doesn't go on the right side but below it. I can see on the demo you're using some css tools propery but I don't see which component / properties does that? Happy to do one custom if it doesn't exist but would like to not overcharge if something exist for that
Same for the integrated table
![image](https://user-images.githubusercontent.com/9129639/182472717-7ed85d8b-2250-4cf5-8eb2-caaa76288a2e.png)

**Comment by timogasda** [comment, marked as answer]
Hello,

Unfortunately, the source code for the demos is not available at the moment, see https://github.com/cloudscape-design/components/discussions/61.

When you have a full-page table like this, i.e., the table is first and main element of the page, you can use the `full-page` variant of the Table component to move it up into the content header of the page. To make this work, you have to do two things:

1. Add `variant="full-page"` and `stickyHeader={true}` to the Table
2. Set `content="table"` on your App layout

You can find an example in our pages: https://github.com/cloudscape-design/components/blob/main/pages/app-layout/with-table.page.tsx

Hope that helps! :)
